### PR TITLE
Feat/profile interests page

### PR DESCRIPTION
**Pull request by themonster2015**
## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/profile/src/components/pages/profile-engagement/interests-page.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <ProfileHeader
+        did={profileData?.did}
+        validAddress={true}
+        background={background}
+        avatar={avatar}
+        name={profileData?.name}
+        ensName={null}
+        viewerIsOwner={isViewer}
+        copyLabel={t('Copy to clipboard')}
+        copiedLabel={t('Copied')}
```
Comment (didd) [inline ui/apps/profile/src/components/pages/profile-engagement/interests-page.tsx]: We can make this props optional on `ProfileHeader` as we aren't actually using them on the interests page.

Comment (themonster2015) [reply ui/apps/profile/src/components/pages/profile-engagement/interests-page.tsx]: I agree

File: ui/design-system-components/src/components/Profile/Header/index.tsx [hunk 2]
```diff
@@ -78,13 +78,14 @@ const Header: React.FC<HeaderProps> = ({
               <DidField
                 did={did.id}
                 isValid={validAddress}
-                copyLabel={copyLabel}
-                copiedLabel={copiedLabel}
+                {...(!copyLabel && !copiedLabel && { copiable: false })}
+                {...(!!copyLabel && { copyLabel: copyLabel })}
+                {...(!!copiedLabel && { copiedLabel: copiedLabel })}
```
Comment (didd) [inline ui/design-system-components/src/components/Profile/Header/index.tsx]: Yen, wouldn't this be more readable if you do:

```
copiable={copyLabel && copiedLabel}
copyLabel={copyLabel}
copiedLabel= {copiedLabel}
```

After all props can be nullable and I am not clear what the fancy object merge achieves other than taking more CPU compute resource :)

**Comment by didd** [comment]
Other than the above comment, the rest of the PR looks fine to me.

File: ui/design-system-components/src/components/Profile/Header/index.tsx (hunk 2 above)
Comment (themonster2015) [reply ui/design-system-components/src/components/Profile/Header/index.tsx]: lol :). Thanks for pointing it out Didd.

**Review by didd — APPROVED** [review]

**Pull request merged**